Detectors/TRD/base/include/TRDBase/TrackletTransformer.h

```diff
@@ -14,6 +14,7 @@
 #include "TRDBase/Geometry.h"
 #include "DataFormatsTRD/Tracklet64.h"
 #include "DataFormatsTRD/CalibratedTracklet.h"
+#include "DataFormatsTRD/Hit.h"
```

Comment:
What is the `Hit` used for?

Reply:
Thanks, it is no longer used, so will remove it.

Reply:
Looks like that header got in there again, could you please remove it?

Reply:
Thanks, sorted